test/periphery/integration/merkle-campaign/ll/claim/claim.t.sol

```diff
@@ -8,45 +8,82 @@ import { MerkleLL_Integration_Shared_Test } from "../MerkleLL.t.sol";
 import { Claim_Integration_Test } from "../../shared/claim/claim.t.sol";
 
 contract Claim_MerkleLL_Integration_Test is Claim_Integration_Test, MerkleLL_Integration_Shared_Test {
+    MerkleLL.Schedule internal schedule;
+
     function setUp() public override(Claim_Integration_Test, MerkleLL_Integration_Shared_Test) {
         super.setUp();
+        schedule = defaults.schedule();
+    }
+
+    modifier whenScheduledStartTimeZero() {
+        _;
     }
 
-    function test_Claim_TimestampsInThePast()
-        external
-        givenCampaignNotExpired
-        givenNotClaimed
-        givenIncludedInMerkleTree
-    {
-        MerkleLL.Schedule memory schedule = defaults.schedule();
-        schedule.startTime = getBlockTimestamp();
+    function test_WhenScheduledCliffDurationZero() external whenScheduledStartTimeZero {
         schedule.cliffDuration = 0;
 
         merkleLL = merkleFactory.createMerkleLL({
-            baseParams: defaults.baseParams(users.admin, dai, defaults.EXPIRATION(), defaults.MERKLE_ROOT()),
+            baseParams: defaults.baseParams("cliff-duration-zero", dai),
             lockupLinear: lockupLinear,
             cancelable: defaults.CANCELABLE(),
             transferable: defaults.TRANSFERABLE(),
             schedule: schedule,
             aggregateAmount: defaults.AGGREGATE_AMOUNT(),
             recipientCount: defaults.RECIPIENT_COUNT()
         });
-        deal({ token: address(dai), to: address(merkleLL), give: defaults.AGGREGATE_AMOUNT() });
 
-        vm.warp(defaults.END_TIME() + 1);
+        // It should create a stream with block.timestamp as start time.
+        // It should create a stream with cliff as zero.
+        _test_Claim({ startTime: getBlockTimestamp(), cliffTime: 0 });
+    }
+
+    function test_WhenScheduledCliffDurationNotZero() external whenScheduledStartTimeZero {
+        merkleLL = merkleFactory.createMerkleLL({
```

Comment:
we don't need to re-deploy this

it is the default `merkleLL` deployed in `MerkleCampaign_Integration_Test`

Reply:
Right.